downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -62,6 +62,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 #### CLI
 
+##### Auto restarting Ballerina Services
+
+`bal run` command has a new flag `--watch` which automatically re-runs Ballerina service projects upon file changes.
```

Comment:
```suggestion
`bal run` command has a new `--watch` option, which automatically re-runs Ballerina service projects upon file changes.
```

Reply:
With https://github.com/ballerina-platform/ballerina-dev-website/pull/9205#discussion_r1705251148, the line changed hence not applicable anymore.